src/main/java/uk/gov/pay/api/filter/ratelimit/RateLimiter.java

```diff
@@ -12,50 +13,51 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RateLimiter.class);
 
-    private final int rate;
-    private final int rateForPost;
+    private int rate;
+    private int rateForPost;
+    private int auditRate;
+    private int perMillis;
+    private int noOfNodes;
     
     private static final String METHOD_POST = "POST";
     
-    private final int auditRate;
-    private final int perMillis;
-    private final Cache<String, RateLimit> cache;
     private final Cache<String, RateLimit> auditCache;
-
-    public RateLimiter(int rate, int rateForPost, int auditRate, int perMillis) {
+    LocalRateLimiter localRateLimiter;
+    RedisRateLimiter redisRateLimiter;
```

Comment:
is there a reason why redisRateLimiter and localRateLimiter aren't private?

Reply:
Changed to private..